tools/AzureRM.BootStrapper/help/about_version_profiles.help.txt

```diff
@@ -0,0 +1,122 @@
+﻿TOPIC
+    about_version_profiles
+
+SHORT DESCRIPTION
+    Version profiles provide a mechanism for managing powershell cmdlets that
+    target  specific versions of azure services supported in different instances
+    of Azure.
+
+LONG DESCRIPTION
+    Different concrete instances of Azure (AzureCloud, AzureChinaCloud, 
+    AzureGermanCloud, AzureUSGovernmentCloud, AzureStack) may have different 
+    versions of Azure services installed, with different capabilities. Azure 
+    Version Profiles provide a mechanism for managing these version differences.
+
+    Each Azure instance has a discoverable set of supported version profiles.
+
+    A user can select a version profile supported by the instances of Azure they
+     target, and this version profile corresponds to versions of the Azure
+    PowerShell  modules. Users can then select these Azure PowerShell module
+    versions and be  confident that their scripts will work when targeting those
+    Azure instances.
+    The AzureRM.Bootstrapper module provides cmdlets to discover, acquire, and
+    use  modules that are appropriate for the azure version profile you are targeting.
+    You can also use Tags in the AzureRM modules to discover profile information
+     for each module version.
+
+    Tags for a Profile use the form VersionProfile:2017-03-09-profile
+    The AzureRM bootstrapper module uses the PowerShell Gallery to install and 
+    load needed modules when you want to target a specific version profile.
+
+EXAMPLES
+Finding appropriate version profiles
+    Use the Get-AzureRmProfile cmdlet to discover available profile versions, 
+    and profile versions supported by an Azure instance.
+
+    Get-AzureRmProfile -ListAvailable
+
+    lists all available version profiles.
+
+    Use-AzureRmProfile -Profile 2017-03-09-profile
+
+    Installs and loads cmdlets for one of the listed profiles.
+
+Targeting all Azure Instances
+    Get-AzureRmProfile
+
+    Lists the profiles that are currently installed on the machine.
+
+    Use-AzureRmProfile -Profile 2017-03-09-profile
```

Comment:
Update this and other instances of 2017-*  to 2019-03-01-hybrid?

Reply:
updated